sklearn/manifold/isomap.py

```diff
@@ -123,6 +132,13 @@ def _fit_transform(self, X):
 
         self.embedding_ = self.kernel_pca_.fit_transform(G)
 
+    @property
+    @deprecated("Attribute training_data_ was deprecated in version 0.20 and "
+                "will be removed in 0.22.")
+    def training_data_(self):
+        check_is_fitted(self, 'nbrs_')
```

Comment:
You should probably write a what's new entry so that you don't forget to include this API change